src/jit/codegenxarch.cpp

```diff
@@ -2654,6 +2640,12 @@ CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
 
         assert(retReg0 != REG_NA && retReg1 != REG_NA);
 
```

Comment:
An issue orthogonal to this issue but related to this method that i happened to observe on lines 2619-2624.  Do we have a Git issue tracking it?  If not please file one.